new-backoffice/tutorials/creating-a-custom-dashboard/adding-functionality-to-the-dashboard.md

```diff
@@ -6,7 +6,7 @@ This page is a work in progress. It will be updated as the software evolves.
 
 ## Overview
 
-This is step 2 in our guide to building a Custom Dashboard. This step continues work on the dashboard we built in [step 1](../creating-a-custom-dashboard.md), but goes further to show how to add functionality and data to our dashboard.
+This is step 2 in our guide to building a Custom Dashboard. This step continues work on the dashboard we built in [step 1](../creating-a-custom-dashboard.md). We will also demonstrate how to add functionality and data.
```

Comment:
Currently, it seems there are 3 very short sentences, and if read out loud might sound a bit robotic. How about leaving the 2nd sentence together like this? 🙈 
```suggestion
This is step 2 in our guide to building a Custom Dashboard. It continues work on the dashboard we built in [step 1](../creating-a-custom-dashboard.md) and will demonstrate how to add functionality and data.
```